OpenIPSL/Data/PowerPlant/Examples/Anderson.mo

```diff
@@ -0,0 +1,82 @@
+within OpenIPSL.Data.PowerPlant.Examples;
+model Anderson
+  "SMIB system to learn how to use the data set records (Anderson book)"
+  extends OpenIPSL.Tests.BaseClasses.SMIB(constantLoad(
+      P_0=powerFlow.powerflow.loads.PL1,
+      Q_0=powerFlow.powerflow.loads.QL1,
+      v_0=powerFlow.powerflow.bus.v3,
+      angle_0=powerFlow.powerflow.bus.A3), gENCLS(
+      P_0=powerFlow.powerflow.machines.PG2,
+      Q_0=powerFlow.powerflow.machines.QG2,
+      v_0=powerFlow.powerflow.bus.v2,
+      angle_0=powerFlow.powerflow.bus.A2));
+  GenerationGroup.Generator2  generator1(
+    P_0=powerFlow.powerflow.machines.PG1,
+    enableP_0=true,
+    Q_0=powerFlow.powerflow.machines.QG1,
+    enableQ_0=true,
+    v_0=powerFlow.powerflow.bus.v1,
+    enablev_0=true,
+    angle_0=powerFlow.powerflow.bus.A1,
+    enableangle_0=true)
+    annotation (Placement(transformation(rotation=0, extent={{-60,-10},{-40,10}})));
+  PFData.PowerFlow powerFlow(redeclare record PowerFlow = PFData.PF00050)
+    annotation (Placement(transformation(extent={{-90,32},{-70,52}})));
+equation
+  connect(generator1.pwPin, GEN1.p)
+    annotation (Line(points={{-39,0},{-30,0}}, color={0,0,255}));
+  annotation (
+experiment(
+      StopTime=20,
+      Tolerance=1e-06),
+    Documentation(info="<html>
+<p>This example demonstrates the use of records with synchronous machine, excitation system, power system stabilizer and turbine-governor model parameters, based on data sets published in the &quot;Power System Control and Stability&quot; book. The generator group for this example includes a <strong>GENROE</strong> machine together with an <strong>ESDC1A</strong> excitation system model, an <strong>IEE2ST</strong> PSS model and an <strong>IEESGO</strong> turbine-governor model.</p>
+<p>Source:</p>
+<ul>
+<li>Anderson, P. M., &amp; Fouad, A. A. (2008). Power system control and stability. John Wiley &amp; Sons. Pages: 566-580.</li>
+</ul>
+</html>"),
+    Diagram(graphics={
+        Rectangle(
+          extent={{-54,86},{98,54}},
+          lineColor={28,108,200},
+          radius=5,
+          lineThickness=0.5,
+          fillColor={255,255,170},
+          fillPattern=FillPattern.Solid),
+        Text(
+          extent={{-48,82},{92,58}},
+          textColor={0,0,0},
+          textString="1. Go to the generator component in the upper-level model.
+ Choose the appropriate parameter data set from the dropdown list
```

Comment:
The description here is wrong since there is no drop-down list for different parameter data sets. The same applies to the other example and the generation group that is pointed to. So this should be double-checked and corrected.

Reply:
The instruction was correct but I forgot to propagate the _GUnitDynamics_ data set to the upper level. I fixed this issue and updated the instructions. Please refer to commit [fafda62](https://github.com/OpenIPSL/OpenIPSL/pull/329/commits/fafda62e0c9180f92a5bb29a1b7576607855113d).

Reply:
Could you please move the description into the infolayer and remove it from the diagram layer. Also please don't use all kinds of fonts, sizes and colours. It just looks untidy. A simple list will do.

Reply:
How about now? See [commit 221f66a](https://github.com/OpenIPSL/OpenIPSL/pull/329/commits/221f66a2423abe478ed02efdc83c0b0e9acdbc12)

Reply:
Sorry, but the instructions are incomplete. I think it is best that you take a colleague (unfamiliar with this data package) and watch him/her whilst they try to follow those instructions. You should then be able to identify missing and imprecise instructions.

Reply:
@maguilerac Have a look at the fixes I've done in https://github.com/OpenIPSL/OpenIPSL/pull/329/commits/0e06d5c617e375cc60559c51a2fec480ceecfb7d to give you an idea on how to create clean documentation with links and also HTML that is more manageable.

Reply:
Ok, @dietmarw! It makes sense to have the <code> tags inside the link text of the hyperlink HTML elements. I have also noticed the indentation changes and the links to OpenIPSL classes. Thank you!